### DISCUSSION TITLE
Is MISO need to connect?

### DISCUSSION THREAD

**Discussion by DamirAS2020** [Bodmer/TFT_eSPI#1590] [Q&A - General] [answered]
Hi,
I have problem with ESP32 GPIO quantity for output.
Tried to declare GPIO34(only for input pin) as MISO.
Working normally.
Then I have disconnected MISO from MC. Working well again.
Is MISO need to connect to ESP32?
320X240 ILI9341.

Thanks,
Damir

**Comment by Bodmer** [comment, marked as answer]
MISO is not needed. You can:
 #define TFT_MISO -1

To flag the pin as unused, or leave out the pin define.

**Reply by DamirAS2020** [reply]
Thanks!
Also reset pin has been connected to 3.3v.
Working well.
